release-notes.hbs.md

```diff
@@ -89,7 +89,9 @@ The following issues, listed by component and area, are resolved in this release
 
 #### <a id='1-8-0-COMPONENT-NAME-ri'></a> v1.8.0 Resolved issues: COMPONENT-NAME
 
-- Resolved issue description.
+#### <a id='1-8-0-scst-store-ri'></a> v1.8.0 Resolved issues: Supply Chain Security Tools - Store
+
+- the expired certificate issue where users need to restart the metadata-store pods when the internal db certificate gets rotated by cert-manager is resolved. Users should no longer see this issue with the default internal db, but the solution does not cover the case of an external db.  
```

Comment:
Do we explain anywhere that we remove tls to the internal db and internal encryption should be handled on the internal network layer?

Reply:
I wasn't sure if we want to explain what we did to fix it, like if that's too much information

Reply:
Should I add that?